test/afterburner/CMakeLists.txt

```diff
@@ -0,0 +1,163 @@
+              COMMAND "python3" "${CMAKE_SOURCE_DIR}/test/energy_scan/mult_and_spectra.py"
+                      "--output_files" ${analysis_output}
+                      "--input_files" "${final_particle_list}"
+              WORKING_DIRECTORY "${test_folder}"
+              COMMENT "Analyzing spectra for ${test_name}  (${i}/${num_runs})."
+              DEPENDS "${CMAKE_SOURCE_DIR}/test/energy_scan/mult_and_spectra.py"
+          )
+    endforeach(i)
+
+    set(analysis_merged_outputs "")
+    set(merged_spectra_fnames "${test_folder}/Merged_spectra/yspectra.txt" ;
+                       "${test_folder}/Merged_spectra/mtspectra.txt" ;
+                       "${test_folder}/Merged_spectra/ptspectra.txt" ;
+                       "${test_folder}/Merged_spectra/v2spectra.txt" ;
+                       "${test_folder}/Merged_spectra/meanmt0_midrapidity.txt" ;
+                       "${test_folder}/Merged_spectra/meanpt_midrapidity.txt" ;
+                       "${test_folder}/Merged_spectra/midrapidity_yield.txt" ;
+                       "${test_folder}/Merged_spectra/total_multiplicity.txt")
+    foreach(j ${merged_spectra_fnames})
+        list(APPEND analysis_merged_outputs ${j})
+    endforeach(j)
+    add_custom_command(OUTPUT ${analysis_merged_outputs}
+        COMMAND "python3" "${CMAKE_SOURCE_DIR}/test/energy_scan/mult_and_spectra.py"
+                "--merge"
+                "--input_files" ${analysis_outputs}
+                "--output_files" ${analysis_merged_outputs}
+        DEPENDS ${analysis_outputs}
+        WORKING_DIRECTORY "${test_folder}"
+        COMMENT "Merge spectra for ${test_name}."
+    )
+
+    set(target afterburner_${test_name})
+    add_custom_target(${target} ALL DEPENDS ${final_particle_lists} ${analysis_merged_outputs} ${plot})
+    #sim target for both processes
+    set(target afterburner_sims_${test_name})
+    add_custom_target(${target} ALL DEPENDS ${final_particle_lists})
+
+    # analysis target for both processes
+    set(target afterburner_analysis_${test_name})
+    add_custom_target(${target} ALL DEPENDS ${analysis_merged_outputs})
+
+    set(all_data_files ${all_data_files};${final_particle_lists} PARENT_SCOPE)
+    set(all_analysis_outputs ${all_analysis_outputs};${analysis_merged_outputs} PARENT_SCOPE)
+endfunction()
+
+set(test_folder ${CMAKE_CURRENT_BINARY_DIR})
+set(plot_names "${test_folder}/Plots/mtspectra_afterburner_RHIC_111.pdf" ;
+               "${test_folder}/Plots/mtspectra_afterburner_LHC_111.pdf" ;)
+
+
+afterburner_test("RHIC") #200GeV
+afterburner_test("LHC")#5TeV
+
+set(plot "")
+foreach(j ${plot_names})
+   list(APPEND plot "${j}")
+endforeach(j)
+if (NOT WITH_EXP_DATA) # plot without experimental data
+  add_custom_command(OUTPUT ${plot}
+      COMMAND "python3" "${CMAKE_SOURCE_DIR}/test/energy_scan/mult_and_spectra_plotter.py"
+              "--theory" ${all_analysis_outputs}
+              "--config" "${CMAKE_CURRENT_BINARY_DIR}/RHIC/data/config.yaml"
```

Comment:
I think there should be a `/RHIC/data/1/config.yaml`

Reply:
Thank you, I will fix this.